packages/utilities/src/index.ts

```diff
@@ -68,6 +68,13 @@ export * from './styled';
 export * from './warn';
 export * from './ie11Detector';
 export * from './getPropsWithDefaults';
+export * from './date-time/components/Calendar.types';
+export * from './date-time/components/CalendarDayGrid';
```

Comment:
Is this the only component we export from utilities? Just seems odd that we're pushing components into here at all, when really they should ideally live in date-time and you can reference them from there.